### PR TITLE
County Names for Score #188

### PR DESCRIPTION
**Pull request by esfoobar-usds**
Work for https://github.com/usds/justice40-tool/issues/188

### CODE REVIEW

**Comment by esfoobar-usds** [comment]
@lucasmbrown-usds ready for review.

File: score/etl/score/etl_county_state_score.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+from etl.base import ExtractTransformLoad
+from utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class CountyStateScoreETL(ExtractTransformLoad):
+    def __init__(self):
+        self.CENSUS_COUNTIES_ZIP_URL = "https://www2.census.gov/geo/docs/maps-data/data/gazetteer/2020_Gazetteer/2020_Gaz_counties_national.zip"
```
Comment (NatHillardUSDS) [inline score/etl/score/etl_county_state_score.py:11]: We'll want to use the 2010 version of this to match the census data we're currently pulling - that's linked to from here: https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.2010.html

**Comment by esfoobar-usds** [comment]
TODO (after meeting with Engineering)

- [ ] - Rename ETL to Score Post Processing - take care of different score output files
- [ ] - Comb through and use 2010 info for everything
- [ ] - Add optimizations mentioned on #302 
- [ ] - After this PR tackle #333 
- [ ] - Clean up per state generation of files (we now use `tilejoin` with the whole us)
- [ ] - Create a ticket to rewrite the local version of mbtiles/mvt generate py so that the Lambda function uses it and users can generate them locally without relying on the GeoPlatform provided US mbtile

**Review by NatHillardUSDS — CHANGES_REQUESTED** [review]
Overall looks great! Only flag would be to use the 2010 version of the data

File: score/etl/score/etl_county_state_score.py (hunk 1 above)
Comment (esfoobar-usds) [reply score/etl/score/etl_county_state_score.py]: Updated

**Review by NatHillardUSDS — APPROVED** [review]
lgtm!
(Ah, sorry, realized just now that there's the above checklist still)

**Pull request merged**